src/utils/XhrNetworkInterceptor.ts

```diff
@@ -150,9 +150,15 @@ export default {
 
               // @ts-ignore
               const _response = this._response;
-              cloneNetwork.requestBody =
-                typeof _response === 'string' ? _response : JSON.stringify(_response);
-              cloneNetwork.responseBody = null;
+              // detect error message when the make xhr requests like using axios
+              if (_response != null && _response.toString().length > 0) {
+                cloneNetwork.errorDomain = typeof _response === 'string' ? _response : '';
+              }
```

Comment:
I think this will cause the `errorDomain` to be empty signaling no error when there has actually been an error in the network request, I think we should set `errorDomain` to `_response` if it's a string and if not we should keep it as `ClientError`, what do you think?

```suggestion
              // Detect a more descriptive error message.
              if (typeof _response === 'string' && _response.length > 0) {
                cloneNetwork.errorDomain = _response;
              }
```